fedora/Dockerfile.armv7hf.tpl

```diff
@@ -33,26 +33,24 @@ RUN curl -SLO "http://resin-packages.s3.amazonaws.com/tini/v$TINI_VERSION/#{TINI
     && chmod +x tini \
     && mv tini /sbin/tini
 
+# Install Systemd
 ENV container docker
 
-RUN systemctl mask \
-        dev-hugepages.mount \
-        sys-fs-fuse-connections.mount \
-        sys-kernel-config.mount \
-        display-manager.service \
-        getty@.service \
-        systemd-logind.service \
-        systemd-remount-fs.service \
-        getty.target \
-        graphical.target \
-        console-getty.service \
-        systemd-vconsole-setup.service
-
-COPY entry.sh /usr/bin/
+# We only want few core services run in the container.
+RUN find /etc/systemd/system \
```

Comment:
We should use `systemctl enable/disable/mask` instead of deleting the files

Reply:
@petrosagg, if we don't delete those files, the logs will be very ugly and can make user confused. For example:
```
18.10.17 02:45:35 (+0700) Failed to install release agent, ignoring: No such file or directory
18.10.17 02:45:35 (+0700) systemd-remount-fs.service: Cannot add dependency job, ignoring: Unit systemd-remount-fs.service is masked.
18.10.17 02:45:35 (+0700) sys-kernel-config.mount: Cannot add dependency job, ignoring: Unit sys-kernel-config.mount is masked.
18.10.17 02:45:35 (+0700) dev-hugepages.mount: Cannot add dependency job, ignoring: Unit dev-hugepages.mount ismasked.
18.10.17 02:45:35 (+0700) systemd-modules-load.service: Cannot add dependency job, ignoring: Unit systemd-modules-load.service is masked.
18.10.17 02:45:35 (+0700) sys-fs-fuse-connections.mount: Cannot add dependency job, ignoring: Unit sys-fs-fuse-connections.mount is masked.
18.10.17 02:45:35 (+0700) kmod-static-nodes.service: Cannot add dependency job, ignoring: Unit kmod-static-nodes.service is masked.
18.10.17 02:45:35 (+0700) getty.target: Cannot add dependency job, ignoring: Unit getty.target is masked.
18.10.17 02:45:35 (+0700) systemd-logind.service: Cannot add dependency job, ignoring: Unit systemd-logind.service is masked.
18.10.17 02:45:35 (+0700) display-manager.service: Cannot add dependency job, ignoring: Unit display-manager.service is masked.
```

Reply:
@nghiant2710 why don't you disable instead of masking then?

Reply:
@petrosagg there're some static services which can't be disabled and they will output errors when systemd starts. 
I've updated this, we should keep masking services as we do now since the above errors seems to be systemd issue (I see the same issue here https://github.com/systemd/systemd/issues/5358). We can explain to users if they ask about these messages.